contributors/design-proposals/seccomp.md

```diff
@@ -133,23 +168,24 @@ Issues:
 * [appc/529](https://github.com/appc/spec/issues/529)
 * [rkt/1614](https://github.com/coreos/rkt/issues/1614)
 
-#### HyperContainer
+### HyperContainer
 
 [HyperContainer](https://hypercontainer.io) does not support seccomp.
 
-### Other platforms and seccomp-like capabilities
-
-FreeBSD has a seccomp/capability-like facility called
-[Capsicum](https://www.freebsd.org/cgi/man.cgi?query=capsicum&sektion=4).
-
-#### lxd
+### lxd
 
 [`lxd`](http://www.ubuntu.com/cloud/lxd) constrains containers using a default profile.
 
 Issues:
 
 * [lxd/1084](https://github.com/lxc/lxd/issues/1084): add knobs for seccomp
 
+### Other platforms and seccomp-like capabilities
+
+FreeBSD has a seccomp/capability-like facility called
+[Capsicum](https://www.freebsd.org/cgi/man.cgi?query=capsicum&sektion=4).
+
+
 ## Proposed Design
 
 ### Seccomp API Resource?
```

Comment:
I think your proposal alters this text about API resources a bit, and I wonder if this ought to be a new proposal (as we have tended to do in the past when making proposals for new additions to a feature).  So, we should probably either add some new text here that continues the narrative about what API resource we're adding, or split your net-new changes into a separate follow-on proposal.